sw/device/lib/dif/dif_padctrl.h

```diff
@@ -0,0 +1,248 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PADCTRL_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PADCTRL_H_
+
+#include <stdbool.h>
+
+// Generated.
+#include "padctrl_regs.h"
```

Comment:
This should be removed from the header, it is sufficient to have it in the source file. Otherwise it adds an additional requirement to the caller libraries to link against the generated register definitions.